src/Umbraco.Web.UI.Client/src/views/relationtypes/edit.controller.js

```diff
@@ -21,6 +21,11 @@ function RelationTypeEditController($scope, $routeParams, relationTypeResource,
 
     function init() {
         vm.page.loading = true;
+        vm.relationsLoading = true;
+
+        vm.changePageNumber = changePageNumber;
+        vm.options = {};
+        vm.options.pageSize = 3;
```

Comment:
This seems to be a testing page size, prob should just be the default :)

Reply:
Yes a page size of 100 seems excessive and I wanted to verify paging was kicking, hence it was set to three for testing.